src/components/openModalDelete.js

```diff
@@ -0,0 +1,19 @@
+import Grid from '@mui/material/Grid';
+import IconButton from '@mui/material/IconButton';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+export default function CustomizedDialogs({children, setOpen}) {
+  const handleClickOpen = () => {
+    setOpen(true);
+  }; 
+  return (
+    <Grid>
+      <IconButton 
+        onClick={handleClickOpen}
+        >
+         <DeleteIcon /> 
+      </IconButton>
+       {children}
+    </Grid>
+  );
+}
```

Comment:
why the file is called in the camel case? If we have all files in one naming convention, don't mix that. Call it DeleteModal instead od openDeleteModal. If you have other components called like that you should find the proper name of each component to describe what component is for.